dockerfiles/init/modules/openshift/files/scripts/wait_until_che_is_available.sh

```diff
@@ -44,7 +44,7 @@ elif [ ${SECONDS} -lt ${end} ]; then
   exit 1
 fi
 
-che_http_status=$(curl -s -o /dev/null -I -w "%{http_code}" "${CHE_API_ENDPOINT}/system/state")
+che_http_status=$(curl -s -o /dev/null -I -w "%{http_code}" "${CHE_API_ENDPOINT}/")
```

Comment:
@garagatyi it's strange. It means that you return `OK` before system is really available.

Reply:
Also OPENSHIFT_TOKEN is a token to access secure endpoint
https://github.com/garagatyi/che/blob/02252426f8e5ef5bb687ca9592573470e9ea866e/dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh#L182-L188

Reply:
I think it is OpenShift token. Does it work in Che?

Reply:
AFAIK we're in the che for openshift script there

Reply:
I think that checking /api is pretty the same check, so it's ok to secure this one

Reply:
it's not. If someone has stopped the system you would think system is ready while it is not.

Reply:
CLI just checks whether the response is 200 or not. System state JSON data are not checked.  It is equal to the check that API responds.

Reply:
so you've found a bug in this script ;-)

Reply:
Well as far as I understand system state API returns 403 in che-multiuser branch now. Permissions are checked but the user is anonymous, so it should not work in any case.